frontend/src/app/sessions/license-indicator/license-indicator.stories.ts

```diff
@@ -0,0 +1,101 @@
+/*
+ * SPDX-FileCopyrightText: Copyright DB InfraGO AG and contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+import { Meta, moduleMetadata, StoryObj } from '@storybook/angular';
+import { BehaviorSubject } from 'rxjs';
+import { PublicLicenseServerWithUsage } from '../../openapi';
+import { LicenseIndicatorComponent } from './license-indicator.component';
+import { LicenseUsageWrapperService } from './license-usage.service';
+
+const meta: Meta<LicenseIndicatorComponent> = {
+  title: 'Session Components/License Indicator',
+  component: LicenseIndicatorComponent,
```

Comment:
```suggestion
  component: LicenseIndicatorComponent,
    decorators: [
    componentWrapperDecorator(
      (story) =>
        `<div class="w-[336px] sm:w-[426px]">
          ${story}
        </div>`,
    ),
  ],
```

Same as in the persistent session story; try to use the same width as in the forntend for a realistic story.